spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -150,24 +158,22 @@ public ManagedChannel stackdriverSenderChannel() {
 				.build();
 	}
 
-	@Bean(REPORTER_BEAN_NAME)
-	@ConditionalOnMissingBean(name = REPORTER_BEAN_NAME)
-	public Reporter<Span> stackdriverReporter(ReporterMetrics reporterMetrics,
-			GcpTraceProperties trace, @Qualifier(SENDER_BEAN_NAME) Sender sender) {
-
+	@Bean(SPAN_HANDLER_BEAN_NAME)
+	@ConditionalOnMissingBean(name = SPAN_HANDLER_BEAN_NAME)
+	public SpanHandler stackdriverSpanHandler(GcpTraceProperties trace, @Qualifier(SENDER_BEAN_NAME) Sender sender) {
 		AsyncReporter<Span> asyncReporter = AsyncReporter.builder(sender)
-				// historical constraint. Note: AsyncReporter supports memory bounds
 				.queuedMaxSpans(1000)
 				.messageTimeout(trace.getMessageTimeout(), TimeUnit.SECONDS)
-				.metrics(reporterMetrics).build(StackdriverEncoder.V2);
```

Comment:
What's replacing this setting?

Reply:
The reporter metrics aren't being pulled in anymore unless you have spring-cloud-sleuth-zipkin. Several iterations I had included an in memory reporter but I just took it out for simplicity. Let me know if I should put it back

Reply:
I'm not even sure what `ReporterMetrics` is for. Maybe it's used in Actuator? It's probably OK to remove it.

Reply:
this would show how many spans or messages were sent or dropped from client POV -> stackdriver (Assuming plumbed to micrometer)

Reply:
Added the `NOOP_METRICS` to allow users to keep reporter metrics but won't incur any memory use if not.